app/templates/components/public/event-map.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="ui grid fluid">
+  <div class="eight wide column loc-map">
+    {{#g-map lat=location.latitude lng=location.longitude zoom=12 as |context|}}
```

Comment:
For this, you should already be able to access the actual event's latitude, longitude and location name as
`event.latitude`, `event.longitude`, `event.locationName`when you pass `model.event` to `public/event-map` like `{{public/event-map event=model.event}}`